core/core.ex

```diff
@@ -48,7 +51,14 @@ defmodule AntikytheraCore do
     System.put_env("ERL_LIBS", new_value)
   end
 
-  defp establish_connections_to_other_nodes(tries_remaining \\ 3) do
+  @connection_retry_interval_ms 5_000
```

Comment:
It seems that the suffix `_in_seconds` is commonly used in this repository.

Reply:
Yes, it is better to follow the convention.
Fixed in https://github.com/access-company/antikythera/pull/168/commits/8e58e0e0acdc0f93411bc2326d5b25ff9abe2e6b